docs/monitoring/monitoring.md

```diff
@@ -1,13 +1,14 @@
 # SnappyData Pulse
 
-SnappyData Pulse is a monitoring system that gives you a high-level overview of the status and performance of the cluster. It provides a simple widget based view, which helps you easily navigate and monitor your cluster.</br>
+SnappyData Pulse is a monitoring system that gives a high-level overview of the status and performance of the cluster.  Pulse provides a simple widget based view, which allows you to easily navigate, visualize, and monitor your cluster. You can monitor the overall status of the cluster as well as of each member in the cluster. 
+All the usage details are automatically refreshed after every 5 seconds, thus you can supervise the live status of your cluster.
 
-To access the SnappyData Pulse, start your cluster and open http:`<leadhost>`:5050/dashboard/ in your web browser. </br>
-`<leadhost>` is the hostname or IP of the lead node in your cluster.
+![Dashboard](../Images/Dashboard.png)
 
-![Dashboard](../Images/monitoring_topnav.png)
+To access SnappyData Pulse, start your cluster and open [http:`<leadhost>`:5050/dashboard/](http:`<leadhost>`:5050/dashboard/) in your web browser.
```

Comment:
Bring this line and note above the dashboard image.

Is it possible to title to an image ?